### ISSUE TITLE
Make the plot_3d_jumps plot rotatable in the browser

### ISSUE THREAD

**Issue by v1kko** [GEMDAT-repos/GEMDAT#83]
I think plotly can be a good candidate for this, but the matplotlib to plotly conversion function seems to be broken for this figure at the moment.
```
  File "/home/vikko/local_projects/GEMDAT/.venv/lib/python3.11/site-packages/plotly/matplotlylib/mplexporter/exporter.py", line 289, in draw_collection
    offset_order = offset_dict[collection.get_offset_position()]
```

The issue is well known and because of a deprecation in matplotlib: this very ugly fix works:
https://github.com/mpld3/mpld3/issues/477
But then again it does not seem to understand more than 2 dimensions, so this is not the way to go.

If we want to do this it is probably best to re-implement it fully in plotly (see also comment below)

**Comment by v1kko** [comment]
I see that there is already an effort for this here: https://github.com/janosh/pymatviz#structure maybe we can somehow contribute there?

**Comment by stefsmeets** [comment]
It's already rotatable in a jupyter notebook if you switch to an interactive backend ('notebook', 'ipympl').

**Comment by stefsmeets** [comment]
Agreed, better to use a javascript/html renderer from the start. 3D in matplotlib is always laggy, whatever backend you use.

I also listed a few dedicated libraries here:
https://github.com/GEMDAT-repos/GEMDAT/issues/23#issuecomment-1619664566

Otherwise, the code for plotting the lattices / atom sites is quite trivial. So any generic renderer will work. It would be nice if it can also show volumes.

**Comment by v1kko** [comment]
I tried it out at least in a jupyter-lab notebook, but I get no interactivity with: 

```
%matplotlib widget
import matplotlib

from pathlib import Path

from gemdat import SimulationData, plot_all, plot
from gemdat.sites import SitesData
from gemdat.io import load_known_material
from gemdat.plots.jumps import plot_jumps_3d

data = SimulationData.from_vasprun(Path('vasprun.xml'))
extra = data.calculate_all(equilibration_steps=1250, diffusing_element='Li')

structure = load_known_material('argyrodite', supercell=(2,1,1))

sd = SitesData(structure)
sd.calculate_all(data=data, extras=extra)
plot_jumps_3d(sites=sd, data=data)
```

**Comment by stefsmeets** [comment]
And `%matplotlib notebook`?

**Comment by v1kko** [comment]
nope, might be my setup, but it really does not want to work

**Comment by v1kko** [comment]
We should fix this in #23 then.
